File: services/location-service/routes/locationRoutes.js
const express = require('express');
const { getSuggestion, checkLocation, giveCoordinates } = require('../controllers/locationController');

const router = express.Router();

// Use the upload.array middleware before calling the requestPhotographer controller
router.get('/getsuggestion', getSuggestion);
router.get('/check', checkLocation);
router.post('/givecoordinates', giveCoordinates)

module.exports = router;

File: services/location-service/routes/locationRoutes.py
from flask import Flask, jsonify, request
from controllers.location_controller import get_suggestion, check_location, give_coordinates

app = Flask(__name__)

# Define the route for checking location
@app.route('/check', methods=['GET'])
def check():
    return check_location()

# Define the route for giving coordinates
@app.route('/givecoordinates', methods=['POST'])
def give_coordinates_route():
    return give_coordinates()

if __name__ == '__main__':
    app.run(debug=True)
